hack/utils/dupissue/dupissue.go

```diff
@@ -0,0 +1,138 @@
+// Copyright 2021 VMware Tanzu Community Edition contributors. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/google/go-github/github"
+	"golang.org/x/oauth2"
+)
+
+const (
+	// TemplateFullPathFilename filename
+	TemplateFullPathFilename string = "issue.template"
+
+	DefaultIssueTemplate string = "This issue was originally filed in the TCE repo, but on a quick inspection, we think this issue probably is best suited for the Tanzu Framework repo.\n\nThe original issue filed in TCE: https://github.com/vmware-tanzu/community-edition/issues/%d\n\n\nThis is a copy and paste from that issue originally filed by @%s:\n\n%s"
+)
+
+func getClientWithEnvToken() (*github.Client, error) {
+	var token string
+	if v := os.Getenv("GITHUB_TOKEN"); v != "" {
+		token = v
+	}
+
+	if token == "" {
+		return nil, fmt.Errorf("token is empty")
+	}
+
+	ctx := context.Background()
+	ts := oauth2.StaticTokenSource(
+		&oauth2.Token{AccessToken: token},
+	)
+	tc := oauth2.NewClient(ctx, ts)
+
+	client := github.NewClient(tc)
+	return client, nil
+}
+
+func getIssue(client *github.Client, issueID int) (*github.Issue, error) {
+	issueGH, _, err := client.Issues.Get(context.Background(), "vmware-tanzu", "community-edition", issueID)
+	if err != nil {
+		fmt.Printf("Issues.Get returned error: %v\n", err)
+		return nil, err
+	}
+
+	return issueGH, nil
+}
+
+func createIssue(client *github.Client, template string, issue *github.Issue) error {
+	newBody := fmt.Sprintf(template, *issue.Number, *issue.User.Login, *issue.Body)
+
+	issueRequest := &github.IssueRequest{
+		Title: issue.Title,
+		Body:  &newBody,
+	}
+	_, _, err := client.Issues.Create(context.Background(), "vmware-tanzu", "tanzu-framework", issueRequest)
+	if err != nil {
+		fmt.Printf("Issues.Create returned error: %v\n", err)
+		return err
+	}
+
+	return nil
+}
+
+func readTemplate() (string, error) {
+	template, err := os.ReadFile(TemplateFullPathFilename)
+	if err == nil {
+		fmt.Printf("Using local template\n")
+		return string(template), nil
+	}
+
+	homedir, err := os.UserHomeDir()
+	if err != nil {
+		fmt.Printf("Failed to get UserHomeDir: %v\n", err)
```

Comment:
Do we want to print out these messages within these calls? Looks like code calling these functions will print out the error again.

Reply:
yea, we dont have to since it's a fairly simple util